kvbc/src/v4blockchain/v4_blockchain.cpp

```diff
@@ -581,6 +584,16 @@ std::optional<categorization::Value> KeyValueBlockchain::getLatest(const std::st
   return latest_keys_.getValue(category_id, key);
 }
 
+std::optional<categorization::Value> KeyValueBlockchain::getFromSnapshot(const std::string &category_id,
+                                                                         const std::string &key,
+                                                                         const BlockId version) const {
+  auto block_id = keys_history_.getUpdateVersionFromSnapshot(category_id, key, version);
+  if (block_id) {
+    return get(category_id, key, block_id.value());
+  }
+  return std::nullopt;
```

Comment:
nit: I would swap the order of returns, return nullopt if !block_id and the happy path is the end of function return

Reply:
ok done